# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/fetch/InternalResponse.h
# Commit: ffcced3e103d
# Full Hash: ffcced3e103d1563abe464e65934ae2d3593db61
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-06-01 09:35:25
# Description:
#   Bug 1833795 - Handle the case that InternalResponse.body/alternativeBody serialization fail. r=dom-worker-reviewers,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D178788
# ==============================================================================

diff -r 1b938c15a507 -r ffcced3e103d dom/fetch/InternalResponse.h
--- a/dom/fetch/InternalResponse.h	Thu Jun 01 01:45:36 2023 +0300
+++ b/dom/fetch/InternalResponse.h	Wed May 31 23:14:11 2023 +0000
@@ -390,6 +390,8 @@
   nsCOMPtr<nsIInputStream> mAlternativeBody;
   nsMainThreadPtrHandle<nsICacheInfoChannel> mCacheInfoChannel;
   bool mCloned;
+  // boolean to indicate the body/alternativeBody will be serialized as a
+  // RemoteLazyInputStream.
   bool mSerializeAsLazy{true};
 
  public:
